livereveal/main.js

```diff
@@ -17,8 +17,15 @@ var config_section = new configmod.ConfigSection('livereveal',
                             {base_url: utils.get_body_data("baseUrl")});
 config_section.load();
 var config = new configmod.ConfigWithDefaults(config_section, {
+    controls: true,
+    progress: true,
+    history: true,
+    width: 1140,
+    height: 855, // 4:3 ratio
```

Comment:
I would like to make this width 1024 and height 768, the deafult ones are really weird values.